libs/vkd3d/command.c

```diff
@@ -2662,10 +2663,16 @@ static void vk_access_and_stage_flags_from_d3d12_resource_state(const struct d3d
                 *stages |= VK_PIPELINE_STAGE_DRAW_INDIRECT_BIT;
                 *access |= VK_ACCESS_INDIRECT_COMMAND_READ_BIT;
 
-                if (device->vk_info.EXT_conditional_rendering)
+                /* D3D12_RESOURCE_STATE_PREDICATION */
+                if (device->device_info.buffer_device_address_features.bufferDeviceAddress)
                 {
-                    *stages |= VK_PIPELINE_STAGE_CONDITIONAL_RENDERING_BIT_EXT;
-                    *access |= VK_ACCESS_CONDITIONAL_RENDERING_READ_BIT_EXT;
+                    *stages |= VK_PIPELINE_STAGE_COMPUTE_SHADER_BIT;
+                    *access |= VK_ACCESS_SHADER_READ_BIT;
+                }
+                else
+                {
+                    *stages |= VK_PIPELINE_STAGE_TRANSFER_BIT;
```

Comment:
I see, this is the fallback when we copy the lower 32-bit predicate?

Reply:
Yes.